evoxbench/database/mosegnas/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+import mosegnas
+
+
+@admin.register(mosegnas.models.MoSegNASResult)
+class MoSegNASAdmin(admin.ModelAdmin):
+    # TODO
```

Comment:
No TODO,
if you really want a TODO, it's ok, but please describe the TODO clearly.